akka-remote/src/test/scala/akka/remote/artery/FlightRecorderSpec.scala

```diff
@@ -372,6 +373,34 @@ class FlightRecorderSpec extends AkkaSpec {
       }
     }
 
+    "create flight recorder file" in {
+      def assertFileIsSound(path: Path) = {
+        Files.exists(path) should ===(true)
+        Files.isRegularFile(path) should ===(true)
+        Files.isWritable(path) should ===(true)
+        Files.isReadable(path) should ===(true)
+      }
+      val fs = Jimfs.newFileSystem()
```

Comment:
I have no idea, (in this case it's just about creating files without messing with the actual filesystem)